MinecraftClient/Resources/config/MinecraftClient.ini

```
@@ -39,6 +39,7 @@ exitonfailure=false                # Disable pauses on error, for using MCC in n
 scriptcache=true                   # Cache compiled scripts for faster load on low-end devices
 timestamps=false                   # Prepend timestamps to chat messages
 autorespawn=false                  # Toggle auto respawn if client player was dead (make sure your spawn point is safe)
+displayrealmsworld=true            # Display available Realms Worlds on start up
```

I suggest changing the setting name because when set to false you cannot join a Realms world at all
```suggestion
minecraftrealms=true               # Enable support for joining Minecraft Realms worlds
```